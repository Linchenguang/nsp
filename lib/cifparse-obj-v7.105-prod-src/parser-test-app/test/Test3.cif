data_061

###############
## CHEM_COMP ##
###############

_chem_comp.id                   '061'
_chem_comp.name
;L-159,061
;
_chem_comp.type                 'inhibitor'
_chem_comp.ndb_synonyms         ?
_chem_comp.formula
;C26 H24 N6 O2
;
_chem_comp.number_atoms_all     58
_chem_comp.number_atoms_nh      34
_chem_comp.formula_weight      452.51
_chem_comp.ndb_formal_charge    0
_chem_comp.rcsb_details          ?

####################
## CHEM_COMP_ATOM ##
####################

loop_
_chem_comp_atom.comp_id
_chem_comp_atom.atom_id
_chem_comp_atom.type_symbol
_chem_comp_atom.model_Cartn_x
_chem_comp_atom.model_Cartn_y
_chem_comp_atom.model_Cartn_z
     061 'N1'        N  .  .  .
     061 'N2'        N  .  .  .
     061 'C5'        C  .  .  .
     061 'N3'        N  .  .  .
     061 'N4'        N  .  .  .
     061 'HN4'       H  .  .  .
     061 'C6'        C  .  .  .
     061 'C7'        C  .  .  .
     061 'C11'       C  .  .  .
     061 'C8'        C  .  .  .
     061 'C12'       C  .  .  .
     061 'C9'        C  .  .  .
     061 'H8'        H  .  .  .
     061 'C10'       C  .  .  .
     061 'H9'        H  .  .  .
     061 'H10'       H  .  .  .
     061 'H11'       H  .  .  .
     061 'C13'       C  .  .  .
     061 'C17'       C  .  .  .
     061 'C14'       C  .  .  .
     061 'H13'       H  .  .  .
     061 'C15'       C  .  .  .
     061 'H14'       H  .  .  .
     061 'C16'       C  .  .  .
     061 'C18'       C  .  .  .
     061 'H16'       H  .  .  .
     061 'H17'       H  .  .  .
     061 'N19'       N  .  .  .
     061 '1H18'      H  .  .  .
     061 '2H18'      H  .  .  .
     061 'C20'       C  .  .  .
     061 'C33'       C  .  .  .
     061 'C21'       C  .  .  .
     061 'N25'       N  .  .  .
     061 'C22'       C  .  .  .
     061 '1H21'      H  .  .  .
     061 '2H21'      H  .  .  .
     061 'C23'       C  .  .  .
     061 '1H22'      H  .  .  .
     061 '2H22'      H  .  .  .
     061 'C24'       C  .  .  .
     061 '1H23'      H  .  .  .
     061 '2H23'      H  .  .  .
     061 '1H24'      H  .  .  .
     061 '2H24'      H  .  .  .
     061 '3H24'      H  .  .  .
     061 'C26'       C  .  .  .
     061 'C27'       C  .  .  .
     061 'C32'       C  .  .  .
     061 'C28'       C  .  .  .
     061 'H27'       H  .  .  .
     061 'C29'       C  .  .  .
     061 'H28'       H  .  .  .
     061 'O30'       O  .  .  .
     061 'C31'       C  .  .  .
     061 'HOU'       H  .  .  .
     061 'H31'       H  .  .  .
     061 'O34'       O  .  .  .

####################
## CHEM_COMP_BOND ##
####################

loop_
_chem_comp_bond.comp_id
_chem_comp_bond.atom_id_1
_chem_comp_bond.atom_id_2
_chem_comp_bond.value_order
_chem_comp_bond.value_dist
_chem_comp_bond.value_dist_esd
     061 'N1'     'N2'     .  .  .
     061 'N1'     'C5'     .  .  .
     061 'N2'     'N3'     .  .  .
     061 'N3'     'N4'     .  .  .
     061 'N4'     'C5'     .  .  .
     061 'N4'     'HN4'    .  .  .
     061 'C5'     'C6'     .  .  .
     061 'C6'     'C7'     .  .  .
     061 'C6'     'C11'    .  .  .
     061 'C7'     'C8'     .  .  .
     061 'C7'     'C12'    .  .  .
     061 'C8'     'C9'     .  .  .
     061 'C8'     'H8'     .  .  .
     061 'C9'     'C10'    .  .  .
     061 'C9'     'H9'     .  .  .
     061 'C10'    'C11'    .  .  .
     061 'C10'    'H10'    .  .  .
     061 'C11'    'H11'    .  .  .
     061 'C12'    'C13'    .  .  .
     061 'C12'    'C17'    .  .  .
     061 'C13'    'C14'    .  .  .
     061 'C13'    'H13'    .  .  .
     061 'C14'    'C15'    .  .  .
     061 'C14'    'H14'    .  .  .
     061 'C15'    'C16'    .  .  .
     061 'C15'    'C18'    .  .  .
     061 'C16'    'C17'    .  .  .
     061 'C16'    'H16'    .  .  .
     061 'C17'    'H17'    .  .  .
     061 'C18'    'N19'    .  .  .
     061 'C18'    '1H18'   .  .  .
     061 'C18'    '2H18'   .  .  .
     061 'N19'    'C20'    .  .  .
     061 'N19'    'C33'    .  .  .
     061 'C20'    'C21'    .  .  .
     061 'C20'    'N25'    .  .  .
     061 'C21'    'C22'    .  .  .
     061 'C21'    '1H21'   .  .  .
     061 'C21'    '2H21'   .  .  .
     061 'C22'    'C23'    .  .  .
     061 'C22'    '1H22'   .  .  .
     061 'C22'    '2H22'   .  .  .
     061 'C23'    'C24'    .  .  .
     061 'C23'    '1H23'   .  .  .
     061 'C23'    '2H23'   .  .  .
     061 'C24'    '1H24'   .  .  .
     061 'C24'    '2H24'   .  .  .
     061 'C24'    '3H24'   .  .  .
     061 'N25'    'C26'    .  .  .
     061 'C26'    'C27'    .  .  .
     061 'C26'    'C32'    .  .  .
     061 'C27'    'C28'    .  .  .
     061 'C27'    'H27'    .  .  .
     061 'C28'    'C29'    .  .  .
     061 'C28'    'H28'    .  .  .
     061 'C29'    'O30'    .  .  .
     061 'C29'    'C31'    .  .  .
     061 'O30'    'HOU'    .  .  .
     061 'C31'    'C32'    .  .  .
     061 'C31'    'H31'    .  .  .
     061 'C32'    'C33'    .  .  .
     061 'C33'    'O34'    .  .  .
