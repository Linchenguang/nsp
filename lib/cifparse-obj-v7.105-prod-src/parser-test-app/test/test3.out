Table TABLE1
Table TABLE2
Table TABLE3
Table TABLE4
Table TABLE5
Table TABLE6
Table TABLE7
Table TABLE8
Table TABLE9

Table TABLE9  Columns: 10  Rows: 100

RowNo  Column-0   Column-1   Column-2   Column-3   Column-4   Column-5   Column-6   Column-7   Column-8   Column-9 
    0   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    1   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    2   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    3   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    4   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    5   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    6   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    7   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    8   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
    9   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   10   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   11   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   12   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   13   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   14   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   15   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   16   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   17   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   18   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   19   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   20   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   21   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   22   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   23   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   24   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   25   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   26   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   27   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   28   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   29   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   30   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   31   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   32   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   33   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   34   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   35   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   36   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   37   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   38   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   39   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   40   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   41   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   42   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   43   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   44   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   45   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   46   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   47   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   48   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   49   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   50   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   51   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   52   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   53   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   54   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   55   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   56   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   57   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   58   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   59   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   60   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   61   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   62   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   63   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   64   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   65   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   66   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   67   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   68   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   69   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   70   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   71   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   72   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   73   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   74   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   75   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   76   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   77   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   78   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   79   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   80   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   81   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   82   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   83   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   84   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   85   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   86   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   87   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   88   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   89   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   90   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   91   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   92   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   93   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   94   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   95   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   96   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   97   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   98   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
   99   Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated    Updated 
